src/puptoo/process/__init__.py

```diff
@@ -38,15 +38,12 @@ def unpacked_archive(msg, remove=True):
     insights archive, and performing cleanup when needed.
     """
     metrics.extraction_count.inc()
-    try:
-        with NamedTemporaryFile(delete=remove) as tf:
-            tf.write(get_archive(msg["url"]))
-            tf.flush()
-            with extract_archive(tf.name) as ex:
-                yield ex
-    finally:
-        metrics.msg_processed.inc()
```

Comment:
These metrics are going to be incremented in the case of a success or failure due to the finally.

Reply:
I see.  In that case, we should probably move `metrics.msg_processed.inc()` to the top of the `extract()` method?   What do you think about that?

Reply:
Also, I think the `metrics.extraction_count.inc()` at the top of the unpacked_archive() method would be calculating the "attempted to extract files".